### ISSUE TITLE
This type cannot unbox to a native string: P6opaque, Failure error

### ISSUE THREAD

**Issue by titsuki** [titsuki/raku-Algorithm-LDA#10]
```
===> Building: Algorithm::LDA:ver<0.0.9>:auth<cpan:TITSUKI>
[Algorithm::LDA] This type cannot unbox to a native string: P6opaque, Failure
[Algorithm::LDA]   in block <unit> at -e line 1
===> Building [FAIL]: Algorithm::LDA:ver<0.0.9>:auth<cpan:TITSUKI>
Aborting due to build failure: Algorithm::LDA:ver<0.0.9>:auth<cpan:TITSUKI> (use --force-build to override)
The command "PERL6LIB=$PWD/lib zef build ." exited with 1.
5.88s$ PERL6LIB=$PWD/lib prove -e perl6 -vr t/
t/01-basic.t .. Must specify something as a path: did you mean '.' for the current directory?
  in sub guess_library_name at /home/travis/.rakudobrew/moar-master/install/share/perl6/core/sources/947BDAB9F96E0E5FCCB383124F923A6BF6F8D76B (NativeCall) line 225
  in method setup at /home/travis/.rakudobrew/moar-master/install/share/perl6/core/sources/947BDAB9F96E0E5FCCB383124F923A6BF6F8D76B (NativeCall) line 285
  in sub lda_create_document at /home/travis/.rakudobrew/moar-master/install/share/perl6/core/sources/947BDAB9F96E0E5FCCB383124F923A6BF6F8D76B (NativeCall) line 594
  in method new at /home/travis/build/titsuki/raku-Algorithm-LDA/lib/Algorithm/LDA/Document.pm6 (Algorithm::LDA::Document) line 10
  in method from-plain at /home/travis/build/titsuki/raku-Algorithm-LDA/lib/Algorithm/LDA/Formatter.pm6 (Algorithm::LDA::Formatter) line 19
  in block <unit> at t/01-basic.t line 14
```

**Comment by titsuki** [comment]
Travis-ci: https://travis-ci.org/titsuki/raku-Algorithm-LDA/jobs/648070169

**Comment by titsuki** [comment]
current one:
```
===> Building: Algorithm::LDA:ver<0.0.9>:auth<cpan:TITSUKI>
[Algorithm::LDA] This type cannot unbox to a native string: P6opaque, Failure
[Algorithm::LDA]   in block <unit> at -e line 1
===> Building [FAIL]: Algorithm::LDA:ver<0.0.9>:auth<cpan:TITSUKI>
Aborting due to build failure: Algorithm::LDA:ver<0.0.9>:auth<cpan:TITSUKI> (use --force-build to override)
The command "PERL6LIB=$PWD/lib zef build ." exited with 1.
6.05s$ PERL6LIB=$PWD/lib prove -e perl6 -vr t/
t/01-basic.t .. Type check failed in binding to parameter '$library'; expected IO::Path but got Slip (Empty)
  in sub guess_library_name at /home/travis/.rakudobrew/moar-master/install/share/perl6/core/sources/947BDAB9F96E0E5FCCB383124F923A6BF6F8D76B (NativeCall) line 216
  in method setup at /home/travis/.rakudobrew/moar-master/install/share/perl6/core/sources/947BDAB9F96E0E5FCCB383124F923A6BF6F8D76B (NativeCall) line 285
  in sub lda_create_document at /home/travis/.rakudobrew/moar-master/install/share/perl6/core/sources/947BDAB9F96E0E5FCCB383124F923A6BF6F8D76B (NativeCall) line 594
  in method new at /home/travis/build/titsuki/raku-Algorithm-LDA/lib/Algorithm/LDA/Document.pm6 (Algorithm::LDA::Document) line 10
  in method from-plain at /home/travis/build/titsuki/raku-Algorithm-LDA/lib/Algorithm/LDA/Formatter.pm6 (Algorithm::LDA::Formatter) line 19
  in block <unit> at t/01-basic.t line 14
```

**Comment by titsuki** [comment]
Fixed by #12
